RFC-008-release-versioning.md

```diff
@@ -0,0 +1,238 @@
+
+The semantic version SHOULD only be updated when the software or document is baselined and shared. It doesn't need to be updated on every author commit during development.
+
+
+### Auto-generated documentation from code
+
+Some documentation is automatically generated from code comments, using tools such as javadoc. In such cases, documentation version numbers should match the code version numbers.
+
+
+### Stand alone documents
+
+Documents which logically should be tracked independently should follow the SemVerDoc standard, as stated.
+
+
+### Documentation set
+
+Often a set of documents are delivered as part of a milestone or release. In these cases, each document should maintain its own version
+number, and then a separate version is assigned to all documents in aggregate.
+
+This should be handled in a similar manner to a linux distribution. Linux distributions typically apply a time based milestone release, and each linux release includes multiple, independently versioned, software packages.
+
+
+### Website
+
+If a website, or a section of the website is versioned, then it should follow the same conventions as a documentation set.
+
+
+### Further reading
+
+-   [Semantic Versioning 2.0.0](https://semver.org/) specification for software.
+    -   [Proposed doc addendum github bug](https://github.com/semver/semver/issues/792)
+-   [Semantic Versioning for Documents and Meaningful Manual Version Control](https://semverdoc.org/) from Nils Tekampe.
+-   [Semantic versioning for docs proposal](http://neelsmith.github.io/2015/12/15/semver/) from Neel Smith.
+-   [Semantic Versioning and Structure for IETF Specifications](https://datatracker.ietf.org/doc/html/draft-claise-semver-02).
+-   This addendum is intended to complement [The Good Docs Project's](https://thegooddocsproject.dev) [RFC 008 Release Versioning proposal](https://github.com/thegooddocsproject/request-for-comment/pull/17).
+-   [Semantic Versioning and Structure for IETF Specifications, draft-claise-semver-02](https://datatracker.ietf.org/doc/html/draft-claise-semver-02).
+-   [Open Geospatial Consortium's Policy Directive 18](https://portal.ogc.org/public_ogc/directives/directives.php) defines use of Major.Minor.Bug-fix versioning for spatial specifications and related documents.
+
+
+## Consequences
+
+This decision will dictate how Releases are communicated to both internal and external parties. Using the Semantic Versioning convention will be familiar to many of the project's most important audiences, namely open source projects (many of which use the same or similar numbering schemes) and technical writers of various levels of experience.
+
+
+## Dependencies
+
+This RFC should be considered together with RFC-006, which describes the high-level planning process to be used for the project's Releases.
+
+
+## Links and prior art
+
+- [Meeting notes from discussion on versioning](https://docs.google.com/document/d/1Oc3hNhJRrXsr-abEmUKAdamC5aKJydtldvpcSK_Hv_E/edit)
+

```

Comment:
Suggestion: Probably worth referencing the point of truth from here too.

```suggestion
- [Semantic versioning addendum for documents. Version 0.1-alpha.4. January 2020](https://docs.google.com/document/d/1gPr_OHoUkg5Wjj3UWYX78ORtTBArSJck4mWrKhWpfPA).

```

Reply:
I made a major mistake and accidentally merged this RFC in prematurely. I've added your suggestion to the body of the new RFC I'm opening.